cmake/leatherman.cmake.in

```diff
@@ -367,3 +367,13 @@ macro(unpack_vendored pkg extracted_dir dir)
         WORKING_DIRECTORY ${PROJECT_BINARY_DIR}/src
         )
 endmacro(unpack_vendored)
+
+# Usage: set_cmake_install_rpath
+#
+# Sets RPATH if not installing to a system library directory
+macro(set_cmake_install_rpath)
```

Comment:
We shouldn't need any RPATH setting here (or in the other projects). By default, CMake adds the directory of any linked libraries to the RPATH. This will cover our INSTALL_PREFIX/lib for pretty much all cases.

That might be a separate change from this work, though